test/cluster_test.go

```diff
@@ -492,12 +497,280 @@ type KubeBenchResultTotals struct {
+        # kubernetes is a component to cover the config file /etc/kubernetes/config that is referred to in the benchmark
+        - kubernetes
+
+      kubernetes:
+        defaultconf: "/etc/kubernetes/config"
+
+      kubelet:
+        cafile:
+          - "/etc/kubernetes/pki/ca.crt"
+          - "/etc/kubernetes/certs/ca.crt"
+          - "/etc/kubernetes/cert/ca.pem"
+          - "/var/snap/microk8s/current/certs/ca.crt"
+        svc:
+          # These paths must also be included
+          #  in the 'confs' property below
+          - "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
+          - "/etc/systemd/system/kubelet.service"
+          - "/lib/systemd/system/kubelet.service"
+          - "/etc/systemd/system/snap.kubelet.daemon.service"
+          - "/etc/systemd/system/snap.microk8s.daemon-kubelet.service"
+        bins:
+          - "hyperkube kubelet"
+          - "kubelet"
+        kubeconfig:
+          - "/etc/kubernetes/kubelet.conf"
+          - "/var/lib/kubelet/kubeconfig"
+          - "/etc/kubernetes/kubelet-kubeconfig"
+          - "/etc/kubernetes/kubelet/kubeconfig"
+          - "/var/snap/microk8s/current/credentials/kubelet.config"
+        confs:
+          - "/var/lib/kubelet/config.yaml"
+          - "/var/lib/kubelet/config.yml"
+          - "/etc/kubernetes/kubelet/kubelet-config.json"
+          - "/etc/kubernetes/kubelet/config"
+          - "/home/kubernetes/kubelet-config.yaml"
+          - "/home/kubernetes/kubelet-config.yml"
+          - "/etc/default/kubelet"
+          - "/var/lib/kubelet/kubeconfig"
+          - "/var/snap/kubelet/current/args"
+          - "/var/snap/microk8s/current/args/kubelet"
+          ## Due to the fact that the kubelet might be configured
+          ## without a kubelet-config file, we use a work-around
+          ## of pointing to the systemd service file (which can also
+          ## hold kubelet configuration).
+          ## Note: The following paths must match the one under 'svc'
+          - "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
+          - "/etc/systemd/system/kubelet.service"
+          - "/lib/systemd/system/kubelet.service"
+          - "/etc/systemd/system/snap.kubelet.daemon.service"
+          - "/etc/systemd/system/snap.microk8s.daemon-kubelet.service"
+        defaultconf: "/var/lib/kubelet/config.yaml"
+        defaultsvc: "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
+        defaultkubeconfig: "/etc/kubernetes/kubelet.conf"
+        defaultcafile: "/etc/kubernetes/pki/ca.crt"
+
+      proxy:
+        optional: true
+        bins:
+          - "kube-proxy"
+          - "hyperkube proxy"
+          - "hyperkube kube-proxy"
+          - "proxy"
+        confs:
+          - /etc/kubernetes/proxy
+          - /etc/kubernetes/addons/kube-proxy-daemonset.yaml
+          - /etc/kubernetes/addons/kube-proxy-daemonset.yml
+          - /var/snap/kube-proxy/current/args
+          - /var/snap/microk8s/current/args/kube-proxy
```

Comment:
Removed `/var/lib/kube-proxy/config.conf` from [default config](https://github.com/aquasecurity/kube-bench/blob/main/cfg/config.yaml#L163)

Reply:
@aidy why was this line removed? (I'm working on the assumption that this patch is based off https://github.com/aquasecurity/kube-bench/pull/809/files)

Reply:
Not removed - but only added to kube-bench 5 hours ago! https://github.com/aquasecurity/kube-bench/pull/820

Reply:
Ah yes! 💡